addons/sale/views/sale_portal_templates.xml

```diff
@@ -295,7 +295,6 @@
                                         If you believe that it is an error, please contact the website administrator.
                                     </div>
                                     <div t-else="" id="payment_method" class="text-start">
-                                        <h3>Pay with</h3>
```

Comment:
Let's not change that in stable. Someone could have xpathed the h3 to remove it; that would break their view :grimacing: